config/params/values.go

```diff
@@ -1,78 +1,11 @@
 package params
 
-import (
-	"fmt"
-
-	"github.com/pkg/errors"
-	fieldparams "github.com/prysmaticlabs/prysm/config/fieldparams"
-	types "github.com/prysmaticlabs/prysm/consensus-types/primitives"
-)
-
 const (
 	EndToEndName        = "end-to-end"
 	EndToEndMainnetName = "end-to-end-mainnet"
 	MainnetName         = "mainnet"
 	MinimalName         = "minimal"
 	PraterName          = "prater"
-)
-
-// KnownConfigs provides an index of all known BeaconChainConfig values.
-var KnownConfigs = map[string]func() *BeaconChainConfig{
```

Comment:
all this code supporting VersionedUnmarshaller got pulled out in favor of the registry.